core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -653,12 +659,24 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/WorkspaceMigrationMetadata'
+        400:
+          description: Some or all workspaces in billing project were ineligible for migration.
```

Comment:
So if you cannot call this method on a billing project where some of the workspaces are not in US multi-region? I would just expect it to ignore those workspaces instead of erroring, as otherwise the method doesn't seem very useful.

Reply:
It will migrate any workspaces that are eligible and the exception will include the list of errors, each of which will include the workspace in question